reload/reload.go

```diff
@@ -0,0 +1,85 @@
+package reload
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/paketo-buildpacks/packit/v2"
+)
+
+var LiveReloadEnabledEnvVar = "BP_LIVE_RELOAD_ENABLED"
+var WatchExecRequirement = packit.BuildPlanRequirement{
+	Name: "watchexec",
+	Metadata: map[string]interface{}{
+		"launch": true,
+	}}
+
+func ShouldRequireWatchexec() (bool, error) {
+	if reload, ok := os.LookupEnv(LiveReloadEnabledEnvVar); ok {
+		if shouldEnableReload, err := strconv.ParseBool(reload); err != nil {
+			return false, fmt.Errorf("failed to parse %s value %s: %w", LiveReloadEnabledEnvVar, reload, err)
+		} else if shouldEnableReload {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+// WatchExecProcessSpec contains information to be translated directly into watchexec arguments
+type WatchExecProcessSpec struct {
+	// WatchDirs will translate into --watch args
+	WatchDirs []string
+
+	// Shell will translate into --shell
+	Shell string
+
+	// Verbose will translate into -v
+	// If Verbose is 0, no -v arg will be provided.
+	// If Verbose is greater than 0, the appropriate number of v's will be added
+	// E.g. Verbose = 2 => -vv
+	// E.g. Verbose = 4 => -vvvv
+	Verbose int
```

Comment:
I'd probably prefer to call this: `VerbosityLevel` - as `Verbose` makes me think of booleans, rather than integers.

Reply:
I like that. I'm not really sure that any buildpacks actually use this, although I have done it once for debugging (https://github.com/paketo-buildpacks/go-build/pull/311#issuecomment-1125042614).